api/openapi-spec/v3/apis__admissionregistration.k8s.io__v1_openapi.json

```diff
@@ -2061,7 +2061,8 @@
                 "$ref": "#/components/schemas/io.k8s.api.admissionregistration.v1.MutatingWebhookConfiguration"
               }
             }
-          }
+          },
+          "required": true
```

Comment:
unrelated to this PR, artifact of kube-openapi bump. None of the openapi-spec schemas have changed.